silver/models/payment_processors/base.py

```diff
@@ -53,6 +53,9 @@ def get_template(self, transaction):
 
         return template
 
+    def was_transaction_initialized(self, transaction, request):
```

Comment:
Maybe it would make sense, to rename this to `handle_transaction_response` or something similar (maybe outcome or feedback instead of response) and have it only update the transaction status based on the request (either made by the client or the payment service provider).

Also a comment in the code, about what the method should do would be nice.